src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -73,13 +73,14 @@ export default class GraphFilter extends React.Component<GraphFilterProps, Graph
     return (
       <>
         <Toolbar>
-          <div className="form-group">
+          <FormGroup style={{ paddingLeft: 0 }}>
```

Comment:
Any `style={{ attribute: value }}` should be considered an anti-pattern just as in normal html framework dev you don't add individual styles but classes. In React it's even easier to inline styles https://zhenyong.github.io/react/tips/inline-styles.html (and with TypeStyle there is even typesafety and css name hashing for global namespace preservation).

Reply:
Alright, sorry, it's difficult to keep up with the speed of change we have with the code styling as well as the actual problem we're trying to solve.  This looked like an easy one but apparently nothing is easy anymore :)  I'll investigate the new way of doing this and generate another PR.